### PR TITLE
Moved `sshd_custom_options` to allow overwriting of sshd options.

### PR DESCRIPTION
**Pull request by ypid**
* Moved it to that position because configuration above should be
  changed using the existing variables.

Fixes: https://github.com/debops/ansible-sshd/pull/19#issuecomment-131523159

### CODE REVIEW

File: templates/etc/ssh/sshd_config.j2 [hunk 1]
```diff
@@ -156,10 +160,6 @@ AllowGroups {{ (sshd_allow_groups|d([]) + sshd_group_allow_groups|d([]) + sshd_h
 
 {% endif %}
 
-{% if sshd_custom_options|d() %}
-# Custom options
-{{ sshd_custom_options }}
-
 {% endif %}
```
Comment (drybjed) [inline templates/etc/ssh/sshd_config.j2]: Forgot about `{% endif %}`.

File: templates/etc/ssh/sshd_config.j2 [hunk 2]
```diff
@@ -66,6 +66,10 @@ MACs {{ sshd_tpl_macs | join(",") }}
 # Privilege Separation is turned on for security
 UsePrivilegeSeparation {{ sshd_privilege_separation }}
 
+{% if sshd_custom_options|d() %}
+# Custom options
+{{ sshd_custom_options }}
+
```
Comment (drybjed) [inline templates/etc/ssh/sshd_config.j2]: Shouldn't this be basically at the top, just after `ansible_managed` to be most effective?

File: templates/etc/ssh/sshd_config.j2 (hunk 1 above)
Comment (ypid) [inline templates/etc/ssh/sshd_config.j2]: Sorry about that. I know why you have an empty line before the endif but it can lead to such problems.

**Comment by ypid** [comment]
Moved to the top.

**Pull request merged**